TAScheduler/AcceptanceTests/test_create_section.py

```diff
@@ -0,0 +1,175 @@
+# PBI #9: Creating Section
+from datetime import datetime
+
+from django.test import TestCase, Client
+
+from TAScheduler.models import Section, Course, Lecture, Lab
+
+
+# Ideas for other tests...
+# 3] check in Database (all fields matching) x2 for [lab+lecture]
+# 4] display .context[] confirmation response
+# 5] check course ID against database (Requirements: Unique + must exist)
+# 6] check section ID against database (Requirements: Unique -doesn't need to exist bc we're creating here-)
+# MISSING "AC4"] Max Course Sections: Check that course1 can't accept any more sections (full course)
+# !OUR DATABASE IS MISSING "hours" FOR SECTION!
+
+# AC1] - Success (Inputs/Course Exists, Unique ID)
+class Create(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.courseList = list()  # holding list of test courses so easier to call later
+        self.secList = list()
+        # Creating 3 Courses: hardcoded "numSec","modality","credits"
+        for i in [1, 2, 3]:
+            testCourse = Course(course_id=i, semester="semester" + str(i), name="name" + str(i), num_of_sections=2,
+                                modality="Remote", credts=3, )
+            testCourse.save()
+            self.courseList = self.courseList.append(testCourse)
+        ### !!!DON'T KNOW IF THIS PART IS NEEDED YET!!!
+        # Creating 3 Sections: hardcoded "meetingTime"
+        for i in [1, 2, 3]:
+            testSection = Section(section_id=i, course=self.courseList[i - 1], location="location" + str(i),
+                                  meeting_time=datetime(2023, 1, 1, 1, 1, 1))
+            testSection.save()
+            self.secList = self.secList.append(testSection)
+        # Assigning course1: lecture + lab ... course2: lecture ... course3: <- No Users assigned.
+        self.lec1 = Lecture(section=self.secList[0], ).save
+        self.lec2 = Lecture(section=self.secList[1]).save
+        self.lab1 = Lab(section=self.secList[2]).save
+        ### ^^^
+        # For user's input ...
+        self.lecInfo = {"section_id": 4, "course": self.courseList[2], "location": "location" + 4,
+                        "meeting_time": datetime(2023, 1, 1, 1, 1, 1), "secType": "Lecture"}
+        self.labInfo = {"section_id": 4, "course": self.courseList[2], "location": "location" + 4,
+                        "meeting_time": datetime(2023, 1, 1, 1, 1, 1), "secType": "Lab"}
+
+    # 1] Creating Unique Lecture w/ Existing course & all inputs
+    # IDK HOW TO SEND IN "LECTURE" AS A DROPDOWN IN POST
+    def test_SuccessfulLecCreation(self):
+        resp = self.client.post("home/managesection/edit", data=self.lecInfo)
```

Comment:
Should be managesection/create not edit.

Reply:
^